overlord/snapstate/backend/export_test.go

```diff
@@ -22,4 +22,6 @@ package backend
 var (
 	AddMountUnit    = addMountUnit
 	RemoveMountUnit = removeMountUnit
+
+	SnapStoreInfoCacheFilename = snapStoreInfoCacheFilename
```

Comment:
```suggestion
	SnapStoreInfoCacheFileName = snapStoreInfoCacheFileName
```

Reply:
hmm...
```
~/.../github.com/snapcore/snapd$ grep -r --include \*.go Filename  | wc -l
79
~/.../github.com/snapcore/snapd$ grep -r --include \*.go FileName  | wc -l
20
~/.../github.com/snapcore/snapd$ grep -r --include \*.go --exclude-dir=vendor/ Filename  | wc -l
75
~/.../github.com/snapcore/snapd$ grep -r --include \*.go --exclude-dir=vendor/ FileName  | wc -l
17
~/.../github.com/snapcore/snapd$ grep -r --include \*.go FileName /snap/go/current/  | wc -l
69
~/.../github.com/snapcore/snapd$ grep -r --include \*.go Filename /snap/go/current/  | wc -l
291
```

Reply:
```
grep --century 15 'heliocentric theory' | wc -l
0
```

Reply:
totally, but you just said "change this", not why, so ...?

Reply:
we tend to use Filename in the codebase more than FileName,  filename is actually a word these days.  Indeed if anything I would go and s/FileName/Filename/ but not for this PR.